sdk/transfer/rule/rule.js

```diff
@@ -0,0 +1,24 @@
+const SubResource = require('../../utils/subResource').SubResource
+
+
+class Rule extends SubResource {
+    /**
+     *
+     * Transfer.Rule object
+     *
+     * @description The Transfer.Rule object modify the behavior of Transfer objects created in your Workspace.
```

Comment:
Melhor deixar essa docstring para o objeto Rules: `The Transfer.Rule object modifies the behavior of Transfer objects when passed as an argument upon their creation.`. Que é o mesmo que está em Python